sdk/guice/v1/pom.xml

```diff
@@ -31,11 +31,6 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>com.google.api.grpc</groupId>
```

Comment:
I don't think this module needs a direct dependency on this. I think it can depend on only the v1 SDK.